lib/reply.js

```diff
@@ -197,6 +198,9 @@ Reply.prototype.headers = function (headers) {
 }
 
 Reply.prototype.code = function (code) {
+  if (undefined === code) {
```

Comment:
I would add a couple more validations to this, like checking if it's a number and it's in the valid range (see rfc).